cmd/bridge/main.go

```diff
@@ -400,6 +411,15 @@ func main() {
 
 		srv.TerminalProxyTLSConfig = serviceProxyTLSConfig
 
+		if *fK8sModeOffClusterGitOps != "" {
+			offClusterGitOpsURL := bridge.ValidateFlagIsURL("k8s-mode-off-cluster-gitops", "https://"+*fK8sModeOffClusterGitOps)
```

Comment:
Sorry I missed this on previous review. You shouldn't prepend `https://`. We expect the URL to be passed already as a full URL with a scheme.

Reply:
Since `https://` is prefixed in the script, `fK8sModeOffClusterGitOps` will just contain `https://` when GitOps isn't setup. Also added a check for that